### ISSUE TITLE
bug: Firefox GNOME Theme plugin doesn't work on 0.3.1+

### ISSUE THREAD

**Issue by lazurit11** [GradienceTeam/Gradience#635]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### What happened?

Plugin doesn't work on versions 0.3.1, 0.3.1-73eb2f6. There is no plugin preferences, and applying theme with plugin enabled doesn't theme Firefox. Meanwhile on 0.3.0 plugin works.

### To Reproduce

1. Go to settings
2. Click on the settings icon of the plugin
3. See "This plugin doesn't have any preferences" alert
4. Go ahead and apply some theme, relog to GNOME
5. See that Firefox has not changed, while GTK apps did

### Screenshots

![image](https://user-images.githubusercontent.com/62574650/195677846-1fb2c7eb-8bee-4e5f-a38c-2ed8156e7bb9.png)

### OS

EndeavourOS

### Version

0.3.0, 0.3.1, 0.3.1-73eb2f6

### Installation method

Flatpak from Flathub

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 0xMRTT** [comment]
The no preferences window is normal. The issue here is that flatpak doesn't have permissions to edit the firefox profile dir. Edit that with Flatseal and retry ?

**Comment by lazurit11** [comment]
Still doesn't work with `filesystem=host`
![image](https://user-images.githubusercontent.com/62574650/195680532-08eb0894-360f-4fde-bdd7-40ed6c1440ad.png)

**Comment by 0xMRTT** [comment]
@daudix-UFO what's the right parameter ?

**Comment by lazurit11** [comment]
Console output of Devel might be useful:

<details>
<summary>Output</summary>

```
[DEBUG] [root] file:///usr/share/backgrounds/gnome/adwaita-d.webp
[DEBUG] [root] adwaita-d.webp
[DEBUG] [root] /usr/share/backgrounds/gnome/adwaita-d.webp
[DEBUG] [yapsy] PluginFileLocator walks (recursively) into directory: /home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/data/gradience/plugins
[DEBUG] [yapsy] PluginFileLocator walks (recursively) into directory: /app/share/gradience/plugins
[DEBUG] [yapsy] firefox_gnome_theme.py is not a valid plugin for strategy info_ext
[DEBUG] [yapsy] PluginFileLocator found a candidate:
    /app/share/gradience/plugins/firefox_gnome_theme.yapsy-plugin
[DEBUG] [yapsy] correct subclass tests failed for: TEMPLATE in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __builtins__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __cached__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __doc__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __file__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __loader__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __name__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __package__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __spec__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [root] setup plugins
WARNING	CSSStylesheet: Unknown @rule found. [1:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [1:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [2:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [2:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [3:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [3:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [4:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [4:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [5:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [5:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [6:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [6:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [7:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [7:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [8:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [8:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [9:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [9:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [10:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [10:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [11:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [11:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [12:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [12:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [13:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [13:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [14:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [14:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [15:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [15:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [16:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [16:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [17:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [17:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [18:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [18:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [19:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [19:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [20:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [20:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [21:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [21:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [22:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [22:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [23:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [23:1: @define-color]
ERROR	CSSUnknownRule: No ending ";" or "}" found: '@define-color headerbar_backdrop_color @window_bg_color;'
[ERROR] [CSSUTILS] CSSUnknownRule: No ending ";" or "}" found: '@define-color headerbar_backdrop_color @window_bg_color;'
WARNING	CSSStylesheet: Unknown @rule found. [24:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [24:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [25:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [25:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [26:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [26:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [27:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [27:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [28:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [28:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [29:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [29:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [30:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [30:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [31:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [31:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [32:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [32:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [33:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [33:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [34:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [34:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [35:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [35:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [36:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [36:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [37:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [37:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [38:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [38:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [39:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [39:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [40:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [40:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [41:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [41:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [42:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [42:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [43:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [43:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [44:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [44:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [45:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [45:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [46:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [46:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [47:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [47:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [48:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [48:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [49:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [49:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [50:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [50:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [51:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [51:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [52:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [52:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [53:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [53:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [54:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [54:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [55:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [55:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [56:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [56:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [57:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [57:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [58:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [58:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [59:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [59:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [60:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [60:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [61:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [61:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [62:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [62:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [63:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [63:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [64:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [64:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [65:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [65:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [66:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [66:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [67:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [67:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [68:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [68:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [69:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [69:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [70:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [70:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [71:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [71:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [72:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [72:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [73:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [73:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [74:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [74:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [75:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [75:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [76:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [76:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [77:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [77:1: @define-color]
WARNING	CSSStylesheet: Unknown @rule found. [78:1: @define-color]
[WARNING] [CSSUTILS] CSSStylesheet: Unknown @rule found. [78:1: @define-color]
[DEBUG] [root] <gradience.utils.preset.Preset object at 0x7f226ebff1c0>
[DEBUG] [root] Kate Dark
[DEBUG] [root] Builder Dark
[DEBUG] [root] Classic Dark
[DEBUG] [root] Gruvbox Dark
[DEBUG] [root] Tango
[DEBUG] [root] Pretty Purple
[DEBUG] [root] Peninsula Dark
[DEBUG] [root] Cobalt Dark
[DEBUG] [root] Builder Light
[DEBUG] [root] normal run
[DEBUG] [root] Kate Dark
[DEBUG] [root] Builder Dark
[DEBUG] [root] Classic Dark
[DEBUG] [root] Gruvbox Dark
[DEBUG] [root] Tango
[DEBUG] [root] Pretty Purple
[DEBUG] [root] Peninsula Dark
[DEBUG] [root] Cobalt Dark
[DEBUG] [root] Builder Light
[DEBUG] [root] reload
[DEBUG] [root] custom_presets: {'user': {}, 'curated': {}, 'official': {'/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/kate-dark': 'Kate Dark', '/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/builder-dark': 'Builder Dark', '/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/classic-dark': 'Classic Dark', '/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/gruvbox-dark': 'Gruvbox Dark', '/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/tango': 'Tango', '/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/pretty-purple': 'Pretty Purple', '/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/peninsula-dark': 'Peninsula Dark', '/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/cobalt-dark': 'Cobalt Dark', '/home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/config/presets/official/builder-light': 'Builder Light'}}
[DEBUG] [root] preset_check: True
[DEBUG] [root] {'Official': 'https://github.com/GradienceTeam/Community/raw/next/official.json', 'Curated': 'https://github.com/GradienceTeam/Community/raw/next/curated.json'}
[DEBUG] [root] apply
[DEBUG] [root] <gradience.utils.preset.Preset object at 0x7f228c1caf20>
[DEBUG] [yapsy] PluginFileLocator walks (recursively) into directory: /home/lazurit/.var/app/com.github.GradienceTeam.Gradience.Devel/data/gradience/plugins
[DEBUG] [yapsy] PluginFileLocator walks (recursively) into directory: /app/share/gradience/plugins
[DEBUG] [yapsy] firefox_gnome_theme.py is not a valid plugin for strategy info_ext
[DEBUG] [yapsy] PluginFileLocator found a candidate:
    /app/share/gradience/plugins/firefox_gnome_theme.yapsy-plugin
[DEBUG] [yapsy] correct subclass tests failed for: TEMPLATE in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __builtins__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __cached__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __doc__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __file__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __loader__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __name__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __package__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [yapsy] correct subclass tests failed for: __spec__ in /app/share/gradience/plugins/firefox_gnome_theme
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/yapsy/PluginManager.py", line 535, in loadPlugins
    is_correct_subclass = issubclass(element, self.categories_interfaces[category_name])
TypeError: issubclass() arg 1 must be a class
[DEBUG] [root] reload plugins
[DEBUG] [root] <yapsy_loaded_plugin_Firefox_GNOME_Theme_1.FirefoxGnomeTheme2Plugin object at 0x7f226ebff9a0>
```

</details>

**Comment by lazurit11** [comment]
Wait, it did work with `0.3.1-73eb2f6` this time!

**Comment by lazurit11** [comment]
But did not with release. I guess you can close this now, or perhaps wait until next release

**Comment by lazurit11** [comment]
FYI: App needs access to `~/.mozilla/firefox` set in Flatseal. Idk if this is ok to do out of the box, but anyway some in-app education for users would be nice

**Comment by 0xMRTT** [comment]
@GradienceTeam/commit since it's builtin, we can add this to the flatpak manifest ?

**Comment by ZechariahB** [comment]
I added this comment here in case someone happens to stumble in a hole in the future.

I tried this plugin in release 0.3.1, but no permission set coerced Gradience to write to customChrome.css. This was due to an issue within 0.3.1's plugin was already fixed in #626. Downloading the fixed version from github and placing them in ``~/.var/app/com.github.GradienceTeam.Gradience/data/gradience/plugins/`` and renaming them as a different plugin solved this. An easier way is to simply update.
